### PR TITLE
Edit user information

### LINKED ISSUE #29: Edit user information - Mobile Phone Portrait
artboard 11

### PR DESCRIPTION
**Pull request by hectorerb**
This PR adds more fields and features to edit user information screen

closed: https://github.com/flyve-mdm/flyve-mdm-ios-agent/issues/29

### CODE REVIEW

File: flyve-mdm-ios/FormInfoCell.swift [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+ * flyve-mdm-ios is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * ------------------------------------------------------------------------------
+ * @author    Hector Rondon
+ * @date      03/08/17
+ * @copyright Copyright © 2017 Teclib. All rights reserved.
+ * @license   GPLv3 https://www.gnu.org/licenses/gpl-3.0.html
+ * @link      https://github.com/flyve-mdm/flyve-mdm-ios
```
Comment (ajsb85) [inline flyve-mdm-ios/FormInfoCell.swift:23]: https://github.com/flyve-mdm/flyve-mdm-ios-agent

**Review by ajsb85 — APPROVED** [review]
A lot of work.

**Review by rafaelje — APPROVED** [review]
Looks good to me 👍

File: flyve-mdm-ios/FormInfoCell.swift (hunk 1 above)
Comment (hectorerb) [reply flyve-mdm-ios/FormInfoCell.swift:23]: yes, I need to update the link

**Pull request merged**